model-optimizer/extensions/front/onnx/random_uniform_ext.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from extensions.ops.random_uniform import AttributedRandomUniform
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.extractor import FrontExtractorOp
+from mo.front.onnx.extractors.utils import onnx_attr, get_onnx_datatype_as_numpy
+from mo.graph.graph import Node
+
+
+class RangeFrontExtractor(FrontExtractorOp):
```

Comment:
Removed ONNX extractor for RandomUniform as in 2022.1 we will use ONNX frontend and this extractor won't be needed.

Reply:
Please, return it back. We will have an option to switch to legacy frontend so it make sense to have it because you have already done some work to enable it.

Reply:
Returned.